### ISSUE TITLE
About DeCCA

### ISSUE THREAD

**Issue by wangsiqinudt** [jameschapman19/cca_zoo#1]
Hi,

Thanks for the excellent toolbox in the first place!

When I tried to run the toolbox ("example_main.py"), there is an error saying that "No module named 'DeCCA'". I checked and found that files about DeCCA are indeed missing in the repo. Would you please confirm that? Are they not released yet?

Thanks and have a nice day!

**Comment by jameschapman19** [comment]
Completely my mistake for pushing an 'unstable' version (has an import of an experimental file!). I've been making a few more changes this week and will probably push a new version later today that I will try to do some basic testing of.

Going forwards I'm intending to add some unit tests and things to avoid this happening it's the first time I've made a python package so I'm learning on the job! XD similarly I'm aware the commenting is patchy so working on that.

Thanks for pointing out 👍

Once I've fixed will message here and close.
